ansible/roles/csm.sbps.lio_config/README.md

```diff
@@ -0,0 +1,37 @@
+csm.sbps.lio_config
+===================
+
+Configure SBPS
+
+Requirements
+------------
+
+None.
+
+Role Variables
+--------------
+
+Dependencies
+------------
+
+Example Playbook
+----------------
+
+```yaml
+    - hosts: Iscsi_Targets
```

Comment:
Should this be `Iscsi_Targets` or `Management_Worker` ?
If this is supposed to run on all NCN workers and nowhere else, then shouldn't we use the latter?